packages/SwingSet/src/vats/comms/dispatch.js

```diff
@@ -10,8 +10,13 @@ import { makeDeliveryKit } from './delivery';
 
 export const debugState = new WeakMap();
 
-export function buildCommsDispatch(syscall) {
-  const state = makeState();
+export function buildCommsDispatch(syscall, _state, _helpers, vatPowers) {
```

Comment:
Oh, I didn't expect `vatParameters` to be delivered as a property of `vatPowers`. But I guess that's what `vatManager/manager-local.js` is currently doing.

Could you fix that? I think it'd be cleaner overall if we change `manager-local.js` from:

```js
    const vatPowers = harden({ ...baseVP, vatParameters, testLog });
    const dispatch = setup(syscall, state, helpers, vatPowers);
```

to:

```js
    const vatPowers = harden({ ...baseVP, testLog });
    const dispatch = setup(syscall, state, helpers, vatPowers, vatParameters);
```

and then `buildCommsDispatch` should be something like:

```js
export function buildCommsDispatch(syscall, _state, _helpers, _vatPowers, vatParameters = {}) {
  const { identifierBase = 0 } = vatParameters;
```

Reply:
A quick grep suggests that we aren't using both `setup` and `vatParameters` anywhere else yet, so I think we can remove it from `vatPowers` without breaking anything. But keep an eye on the tests in case I missed something.

Reply:
done